app/javascript/src/PaymentGateways/braintree/BraintreeForm.jsx

```diff
@@ -57,14 +57,12 @@ const BraintreeForm = ({
   }, [braintreeNonceValue])
 
   const get3DSecureError = (response) => {
-    if (
-      response.name === 'BraintreeError' ||
-      response.threeDSecureInfo.status !== 'authenticate_successful' ||
-      response.threeDSecureInfo.status !== 'authenticate_attempt_successful'
-    ) {
-      return CC_ERROR_MESSAGE
-    }
-    return null
+    return (
+      !response.threeDSecureInfo ||
```

Comment:
@guicassolato 
You were right, we don't need to verify `response.name === 'BraintreeError'`, but we need to verify `response.threeDSecureInfo`

- With unsupported cards, it doesn't try to do the 3Dsecure verification. The response will include `response.name`, but not `response.threeDSecureInfo` object at all.
- When the card is supported, it calls 3DSecure `verifyCard`, and response will have `threeDSecureInfo.status`